spec/elf/EverParseEndianness.h

```diff
@@ -0,0 +1,110 @@
+/*++
+
+Copyright (c) Microsoft Corporation
+
+Module Name:
+
+EverParseEndianness.h
+
+Abstract:
+
+This is an EverParse-related file to read integer values from raw
+bytes.
+
+Authors:
+
+nswamy, protz, taramana 5-Feb-2020
+
+--*/
+/* This is a hand-written header that selectively includes relevant bits from
+ * kremlib.h -- it has to be updated manually to track upstream changes. */
```

Comment:
Can these be pulled in from a submodule instead of copied into our repo?

Reply:
Generated file.

Reply:
https://github.com/microsoft/ebpf-for-windows/issues/834